estate/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# license
+
```

Comment:
```suggestion
```
The `# -*- coding: utf-8 -*-` line was necessary in Python 2 to specify the source file's character encoding because Python 2 defaulted to ASCII.

In Python 3, which Odoo now uses, UTF-8 is the default source code encoding. Therefore, this declaration is redundant and can be safely removed. This comment applies to a few other files in the PR.